hackathon/views.py

```diff
@@ -188,3 +190,46 @@ def delete_hackathon(request, hackathon_id):
 
     messages.success(request, f'{hackathon.display_name} has been successfully deleted!')
     return redirect("hackathon:hackathon-list")
+
+class HackathonDetailView(DetailView):
+    """Renders a page with Hackathon details."""
+    model = Hackathon
+    context_object_name = "hackathon"
+
+
+def ajax_enroll_toggle(request):
+    """Swaps between being enrolled as a judge and unenrolling."""
+    # Checks to make sure the user has the permissions to enroll
+    user = request.user
+    data = {}
+    if (request.method == "POST"):
```

Comment:
I don't think the parenthesis here are needed.

Reply:
done